### ISSUE TITLE
Explorer Compact Folders config setting no longer applies to top level folders in workspace

### ISSUE THREAD

**Issue by xploSEoF** [microsoft/vscode#135003]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.61.0
- OS Version: Windows_NT x64 10.0.19043 (using WSL1)

Steps to Reproduce:

1. Open an existing workspace with multiple top level folders, or create one
2. Notice the Explorer now opens all top level folders

If this is a desirable feature, perhaps a separate config setting for applying Compact Folders setting to the top level folders, too.

**Comment by ccleve** [comment]
Same here. This started happening yesterday. It's probably a new issue in this month's release.

"Collapse Folders in Explorer" does not work if the cursor is on a file. You have to select a top-level folder in the Explorer and then it works.

**Comment by ccleve** [comment]
Yes, this does appear to be an exact duplicate of #134665 Sorry I missed that one.
